Wrappers/Python/ccpi/optimisation/operators/GradientOperator.py

```diff
@@ -409,8 +380,10 @@ def adjoint(self, x, out=None):
             return_val = True
 
         ndout, out_p = Gradient_C.datacontainer_as_c_pointer(out)
+        
+        tmp = x/self.voxel_size_order
```

Comment:
This should skip the divide if voxel size equals 1

Reply:
done